Cargo.toml

```diff
@@ -24,6 +23,7 @@ serde = "1.0.195"
 
 [dev-dependencies]
 tokio = { version = "1.28.0", features = ["full"] }
+anyhow = "1.0.70"
```

Comment:
does this need to be a dev-dep?

Reply:
no i just made it one because we aren't using it outside of tests